lib/catalog/widgets/sized_box.dart

```diff
@@ -14,11 +15,15 @@ class SizedBoxPropertyExplorer extends StatelessWidget {
       builder: (provider) {
         final height = provider.heightField();
         final width = provider.widthField();
-        return SizedBox(
-          height: height,
-          width: width,
-          child: ColoredBox(
-            color: Theme.of(context).primaryColor,
+        final alignment = provider.alignmentField(id: "alignment", title: "Alignment");
```

Comment:
you can use Single Widget Container to get all style options , rather than align , size and box color .